index.html

```diff
@@ -47,6 +48,157 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
+        <li><a href="#" class="anc card-a3">Javscript</a></li>
+        <li><a href="#" class="anc card-a4">html</a></li>
+      </ul>
+      <a href="#" class="tag">
+        <button type="button" class="btn btn-success see-button">See Project</button>
+      </a>
+    </div>
+
+    <div class="card">
+      <div class="card-image car"></div>
+      <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+      <ul class="alist">
+        <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+        <li><a href="#" class="anc card-a2">CSS</a></li>
+        <li><a href="#" class="anc card-a3">Javscript</a></li>
+        <li><a href="#" class="anc card-a4">html</a></li>
+      </ul>
+      <a href="#" class="tag">
+        <button type="button" class="btn btn-success see-button">See Project</button>
+      </a>
+    </div>
+
+ <!-- About me -->
+    <div class="about-me">
+      <h1 class="intro">About Me</h1>
+      <p class="about-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+don’t hestiate to contact me.</p>
+      <img src="images/about.svg" alt="">
+      <button type="button" class="btn btn-success resume-button">Get My Resume</button>
+    </div>
+
+    <div class="card">
+      <div class="card-image sphere">
+        <img src="images/sphere.svg" class="sp-image" alt="">
+      </div>
+      <h2 class="card-text2">Languages</h2>
+      <ul class="alist">
+        <li><a href="#" class="anc2">Javascript</a></li>
+        <li><a href="#" class="anc2">Ruby</a></li>
+        <li><a href="#" class="anc2">Html</a></li>
+        <li><a href="#" class="anc2">CSS</a></li>
+      </ul>
+
+    </div>
+
+    <div class="card">
+      <div class="card-image diamond">
+        <img src="images/diamond.svg" class="dm-image" alt="">
+      </div>
+      <h2 class="card-text2">Frameworks</h2>
+      <ul class="alist">
+        <li><a href="#" class="anc2">Bootstrap</a></li>
+        <li><a href="#" class="anc2">Ruby</a></li>
+        <li><a href="#" class="anc2">RSpec</a></li>
+        <li><a href="#" class="anc2">Capybara</a></li>
+        <li><a href="#" class="anc2">Selenium</a></li>
+      </ul>
+
+    </div>
+
+    <div class="card">
+      <div class="card-image light">
+        <img src="images/light.svg" class="lt-image" alt="">
+      </div>
+      <h2 class="card-text2">Skills</h2>
+      <ul class="alist">
+        <li><a href="#" class="anc2">Codekit</a></li>
+        <li><a href="#" class="anc2">Github</a></li>
+        <li><a href="#" class="anc2">Codepen</a></li>
+        <li><a href="#" class="anc2">Gitlab</a></li>
+        <li><a href="#" class="anc2">Terminal</a></li>
+      </ul>
+
+   </div>
+
+   <!-- get in touch -->
+   <div class="contact">
+     <h2 class="contact-info">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
```

Comment:
:orange_circle: [OPTIONAL] I think the proper semantic tag here is p and not h2 tag